fastcov.py

```diff
@@ -60,6 +60,133 @@ def format(self, record):
         log_message = super(FastcovFormatter, self).format(record)
         return "[{:.3f}s] {}".format(stopwatch(), log_message)
 
+class DiffParseError(Exception):
+    pass
+
+class DiffParser(object):
+    def _refinePaths(self, diff_metadata, diff_base_dir):
+        diff_metadata.pop('/dev/null', None)
+        diff_metadata.pop('', None)
+        for key, value in diff_metadata.copy().items():
+            diff_metadata.pop(key)
+            #sources without added lines will be excluded
+            if value:
+                newpath = os.path.join(diff_base_dir, key) if diff_base_dir else os.path.abspath(key)
+                newpath = newpath.replace('\\', '/')
```

Comment:
Can you give an example where backslashes in a path would need to be converted to forward slashes?

Reply:
That was some kind of cross-platform work with diff file filtering, but while I was writing unittest on Windows, completely forgot that this tool is not designed to be run on windows. So I'm deleting it